pkg/controller/bundlec/controller.go

```diff
@@ -66,23 +77,50 @@ type Controller struct {
 }
 
 // Prepare prepares the controller to be run.
-func (c *Controller) Prepare(crdInf cache.SharedIndexInformer, resourceInfs map[schema.GroupVersionKind]cache.SharedIndexInformer) {
+func (c *Controller) Prepare(crdInf cache.SharedIndexInformer, resourceInfs map[schema.GroupVersionKind]cache.SharedIndexInformer) error {
 	c.crdContext, c.crdContextCancel = context.WithCancel(context.Background())
 	crdInf.AddEventHandler(&crdEventHandler{
 		controller: c,
 		watchers:   make(map[string]watchState),
 	})
-
+	deploymentInf := resourceInfs[apps_v1.SchemeGroupVersion.WithKind("Deployment")]
```

Comment:
nit: I know that k/k monorepo doesn't have constants for resource kinds, but we could define our own constants and use them?

Reply:
Yes. A separate PR that updates all places with constants would be nice.

Reply:
https://github.com/atlassian/smith/issues/403